### DISCUSSION TITLE
is `fop.response()` similar with `fop.jacobian().mult(model)`?

### DISCUSSION THREAD

**Discussion by yudhastyawan** [gimli-org/pyGIMLi#287] [Q&A]
Hi, I am yudha styawan as a student in Taiwan. Thank you and all contributors for creating a useful package for us, especially in geophysical problems. I just submitted my question in issue https://github.com/gimli-org/gimli/issues/286. In there, I just wonder about is `fop.response()` similar with `fop.jacobian().mult(model)` or not? Thank you for your kind response.

**Comment by halbmy** [comment]
At least in case of a ray approximation solution (as we are doing with Dijkstras algorithm), `fop.jacobian().mult(model)` should be identical to `fop.response()` and it seems to be the case in 2D for different models (we have not much experience with 3D).

If it comes to alternative solutions like considering the Fresnel volume by fat rays (Jordi et al. 2016), the two will be different. If we don't find the bug, we would need a synthetic test case to see whether the incorrect Jacobian is too bad or just decreases convergence a bit.

Jordi et al. (2016): Frequency-dependent traveltime tomography using fat rays: application to near-surface seismic imaging, J. Appl. Geoph. 131, 202-213, http://dx.doi.org/10.1016/j.jappgeo.2016.06.002

**Reply by yudhastyawan** [reply]
Thank you for your response and suggestions here and on the Issues page. I would check the reference soon to see the method being used. Because we have different results between `fop.jacobian().mult(model)` and `fop.response(model)`, which one has implemented the method provided in the reference, or both of them? I also have commented on the Issues page. Thank you.
